src/components/filter/FilterPanel.tsx

```diff
@@ -72,7 +72,11 @@ const FilterPanel = (props: Props) => {
 	}, [showFilter]);
 
 	useOutsideClickListener(filterBaseRef, (e: MouseEvent) => {
-		if (!filterButtonRef.current?.contains(e.target as Element)) {
+		const target = e.target as Element;
+		if (target.classList.contains('filter-history__item')) {
+			e.stopImmediatePropagation();
```

Comment:
It's not here where you need to stop event propagation, but you should do it inside its inner modal which is FiltersHistory